src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3729,6 +3729,13 @@ bool AdaptDispatch::DoFinalTermAction(const std::wstring_view string)
     return false;
 }
 
+// We literally only implement the xterm.js sequences that are aliases for the
+// final term ones. Just implement exactly the same.
+bool AdaptDispatch::DoXtermJsAction(const std::wstring_view string)
+{
+    return DoFinalTermAction(string);
+}
```

Comment:
Instead of adding another interface method, we can also do this:
```cpp
case OscActionCodes::FinalTermAction:
case OscActionCodes::XtermJsAction:
    success = _dispatch->DoFinalTermAction(string);
    break;
```

While I understand that they aren't ideologically the same thing, they're semantically identical at the moment. As such I believe that the split has no "tangible" benefit right now and I think it would be better without it (in other similar places as well for the same reason).

Reply:
That would make this review so so much shorter...!